db/revtree.go

```diff
@@ -724,12 +724,15 @@ func (tree RevTree) RenderGraphvizDot() string {
 func (tree RevTree) getHistory(revid string) ([]string, error) {
 	maxHistory := len(tree)
 
+	var totalBytesForHistory int
 	history := make([]string, 0, 5)
 	for revid != "" {
 		info, err := tree.getInfo(revid)
 		if err != nil {
 			break
 		}
+		revBytes := len([]byte(revid))
+		totalBytesForHistory += revBytes
```

Comment:
totalBytesForHistory looks to be unused, I must've missed from clean up form earlier implementation of the rev cache work. I have filed [CBG-4277](https://jira.issues.couchbase.com/browse/CBG-4277) for removing this off main. We can either merge s is then backport the ticket on main or manually remove this now before merging?

Reply:
I'm happy to merge that and do another cherry-pick into this PR :)